instrumentation-core/spring/spring-boot-autoconfigure/src/test/java/io/opentelemetry/instrumentation/spring/autoconfigure/exporters/JaegerSpanExporterAutoConfigurationTest.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.instrumentation.spring.autoconfigure.exporters;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+
+import io.opentelemetry.exporters.jaeger.JaegerGrpcSpanExporter;
+import io.opentelemetry.instrumentation.spring.autoconfigure.TracerAutoConfiguration;
+import io.opentelemetry.instrumentation.spring.autoconfigure.exporters.jaeger.JaegerSpanExporterAutoConfiguration;
+import io.opentelemetry.instrumentation.spring.autoconfigure.exporters.jaeger.JaegerSpanExporterProperties;
+import java.time.Duration;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.autoconfigure.AutoConfigurations;
+import org.springframework.boot.test.context.runner.ApplicationContextRunner;
+
+/** Spring Boot auto configuration test for {@link JaegerGrpcSpanExporter}. */
+public class JaegerSpanExporterAutoConfigurationTest {
+
+  private final ApplicationContextRunner contextRunner =
+      new ApplicationContextRunner()
+          .withConfiguration(
+              AutoConfigurations.of(
+                  TracerAutoConfiguration.class, JaegerSpanExporterAutoConfiguration.class));
+
+  @Test
+  public void should_initialize_JaegerGrpcSpanExporter_bean_when_exporters_are_ENABLED() {
+    this.contextRunner
+        .withPropertyValues("opentelemetry.trace.exporters.jaeger.enabled=true")
+        .run(
+            (context) -> {
+              assertNotNull(
+                  "Application Context contains JaegerGrpcSpanExporter bean",
+                  context.getBean("otelJaegerSpanExporter", JaegerGrpcSpanExporter.class));
+            });
+  }
+
+  @Test
+  public void should_initialize_JaegerGrpcSpanExporter_bean_with_property_values() {
```

Comment:
I think this test is only really checking `JaegerGrpcSpanExporterProperties`, not that the properties are actually applied, which is probably not really possible. So can update the naming